multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/Messages.java

```diff
@@ -69,6 +69,7 @@
     // ERROR log messages
     public static final String COULD_NOT_COMPUTE_ORG_AND_SPACE = "Could not find org and space for space ID \"{0}\"";
     public static final String NO_BUILDS_FOUND_FOR_PACKAGE = "No builds found for package \"{0}\"";
+    public static final String ERROR_ASYNC_OPERATION_SERVICE_BROKER_FAILED = "Async operation of service broker \"{0}\" has been failed with \"{1}\"";
```

Comment:
I don't think the operations are "of" the service brokers. That would imply the brokers _themselves_ are managing the operations. 
And "has been failed" doesn't sound grammatically correct.
My suggestion is either:
"Service broker {0} async operation failed with {1}"
_or_
"Async operation for service broker {0} failed with {1}"